### ISSUE TITLE
Add a column for downloads per month?

### ISSUE THREAD

**Issue by cboettig** [ropensci/dashboard#9]
Might be cool to show downloads per month or something in addition to total downloads

**Comment by karthik** [comment]
Yes absolutely. I can add that to maybe the tooltip with all these additional bits.
